src/electron/go_vpn_tunnel.ts

```diff
@@ -17,7 +17,7 @@ import {powerMonitor} from 'electron';
 import {platform} from 'os';
 import {promisify} from 'util';
 
-import {ShadowsocksConfig} from '../www/app/config';
```

Comment:
Do you think "shadowsocks_session_config" should be a part of the model, actually?